### ISSUE TITLE
display_psf(..., markcentroid=True) supplies extra arg to fwcentroid

### ISSUE THREAD

**Issue by mperrin** [spacetelescope/poppy#175]
<a href="https://github.com/josePhoenix"><img src="https://avatars3.githubusercontent.com/u/172886?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [josePhoenix](https://github.com/josePhoenix)**
_Wednesday Jul 27, 2016 at 20:24 GMT_
_Originally opened as https://github.com/mperrin/poppy/issues/175_

----

The `threshold` argument to `measure_centroid` is getting passed through to `fwcentroid()`, but `fwcentroid()` is not expecting it.

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-162-a137231ed340> in <module>()
----> 1 poppy.display_psf(hdul, markcentroid=True)

/Users/jlong/software/poppy/poppy/utils.py in display_psf(HDUlist_or_filename, ext, vmin, vmax, scale, cmap, title, imagecrop, adjust_for_oversampling, normalize, crosshairs, markcentroid, colorbar, colorbar_orientation, pixelscale, ax, return_ax, interpolation)
    248     if markcentroid:
    249         _log.info("measuring centroid to mark on plot...")
--> 250         ceny, cenx = measure_centroid(HDUlist, ext=ext, units='arcsec', relativeto='center', boxsize=20, threshhold=0.1)
    251         ax.plot(cenx, ceny, 'k+', markersize=15, markeredgewidth=1)
    252         _log.info("centroid: (%f, %f) " % (cenx, ceny))

/Users/jlong/software/poppy/poppy/utils.py in measure_centroid(HDUlist_or_filename, ext, slice, boxsize, verbose, units, relativeto, **kwargs)
    813         image = image[slice, :, :]
    814 
--> 815     cent_of_mass = fwcentroid(image, halfwidth=boxsize, **kwargs)
    816 
    817     if verbose:

TypeError: fwcentroid() got an unexpected keyword argument 'threshhold'
```

**Comment by mperrin** [comment]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mperrin](https://github.com/mperrin)**
_Monday Sep 26, 2016 at 17:14 GMT_

----

@josePhoenix  any chance you could take a shot at debugging this one? Presumably is a matter of just more careful handling of **kwargs.

**Comment by mperrin** [comment]
<a href="https://github.com/josePhoenix"><img src="https://avatars3.githubusercontent.com/u/172886?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [josePhoenix](https://github.com/josePhoenix)**
_Friday Oct 28, 2016 at 16:42 GMT_

----

Simpler than that, even... upon investigation, `threshhold` and `threshold` are not the same 😄

This comes under the heading of "testing the plotting code", I think, so I'm just going to fix the typo and not add a regression test.

**Comment by mperrin** [comment]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mperrin](https://github.com/mperrin)**
_Monday Apr 10, 2017 at 14:40 GMT_

----

This typo was fixed last year in d127fc3. Closing issue.
